client/src/components/layouts/components/guide/guideFooter.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from '../../../../components/helpers';
+
+const propTypes = {
+  path: PropTypes.string
+};
+
+class GuideFooter extends React.Component {
+  render() {
+    const { path } = this.props;
+    const slug = path.split('/guide')[1];
+    const urlEnding = slug !== '/' ? 'index.md' : '';
+    const fileUrl = `https://github.com/freeCodeCamp/freeCodeCamp/blob/master/guide/english${slug}${urlEnding}`;
```

Comment:
We need a more elegant solution than hard coding this here? You should create this slugs dynamically. See [more details here](https://www.gatsbyjs.org/docs/node-apis/)